pytest/tests/shardnet/scripts/create_account.sh

```diff
@@ -5,5 +5,5 @@ account_id=$(grep account_id /home/ubuntu/.near/shardnet/validator_key.json | aw
 mkdir -p /home/ubuntu/.near-credentials/shardnet/
 echo "{\"account_id\":\"near\",\"public_key\":\"$1\",\"private_key\":\"$2\"}" > /home/ubuntu/.near-credentials/shardnet/near.json
```

Comment:
```suggestion
printf '{"account_id":"near","public_key":"%s","private_key":"%s"}' \
    "${1:?}" "${2:?}" > /home/ubuntu/.near-credentials/shardnet/near.json
```
Also do `set -eux` at the top.